1.9/networking/service-and-pod-networking.md

```diff
@@ -0,0 +1,422 @@
+---
+post_title: Service and Pod Networking
+menu_order: 0.5
+---
+
+This document describes how to specify networking modes and assign ports in the service definition for a DC/OS Service you are creating.
+
+# VIPs
+
+We recommend using [virtual addresses (VIPs)](/docs/1.9/usage/service-discovery/virtual-ip-addresses/) to make ports management easier. VIPs map traffic from a single virtual address to multiple IP addresses and ports. They simplify inter-app communication and implement a reliable service-oriented architecture.
+
+# Networking Modes
+
+DC/OS services and pods declare `networks` the same way. Three modes of networking are supported:
+
+## `host` Networking
+
+In `host` networking, an application shares the network namespace of the Mesos agent process, typically the host network namespace.
+
+## `container` Networking
+
+In container networking, an application should be allocated its own network namespace and IP address;
+Mesos network isolators are responsible for providing backend support for this.
+When using the Docker containerizer, this translates to a Docker "user" network. You can name your container network in the `networks.container` parameter of your service definition. If you do not provide a name for your container network, it will have the default container network name, `DEFAULT NAME`.
+
```

Comment:
I don't understand this part well enough to know if all of this is relevant for DC/OS users.